rviz_common/src/rviz_common/yaml_config_reader.cpp

```diff
@@ -91,35 +78,17 @@ void YamlConfigReader::readYamlNode(Config & config, const YAML::Node & yaml_nod
   switch (yaml_node.Type() ) {
     case YAML::NodeType::Map:
       {
-#ifdef RVIZ_HAVE_YAMLCPP_05
-        for (YAML::const_iterator it = yaml_node.begin(); it != yaml_node.end(); ++it)
-#else
-        for (YAML::Iterator it = yaml_node.begin(); it != yaml_node.end(); ++it)
-#endif
-        {
+        for (YAML::const_iterator it = yaml_node.begin(); it != yaml_node.end(); ++it) {
```

Comment:
Did you intentionally move the brace to the same line here, and in the other change below? Looks different from the style used in the rest of the file. Hopefully it will be caught by a linter.